.gitignore

```diff
@@ -1,5 +1,5 @@
 node_modules
 *.log
 dist
-coverage
```

Comment:
Weird, this actually works for ignoring the directory in git, but vscode wouldnt recognise it (I'd still get search results from this dir).